tasks/msi.py

```diff
@@ -71,7 +71,7 @@ def _get_vs_build_command(cmd, vstudio_root=None):
     return cmd
 
 
-def _get_env(ctx, major_version='7', release_version='nightly'):
+def _get_env(ctx, major_version='7', release_version='a7'):
```

Comment:
Couldn't `release_version` be inferred from `major_version`?

Reply:
Since the plan is to follow this up by another PR to drop the need for multiple release entries and this won't be needed, I feel inclined to leave this as is to make the change as mechanical as possible.

Reply:
Fair enough :)